tests/test_plugin.py

```diff
@@ -63,6 +63,23 @@ def test_build_plugin_failure(docker_tasker):
         results = runner.run()
     assert workflow.build_process_failed is True
 
+@pytest.mark.parametrize('required', [
+    True,
+    False,
+])
+def test_not_required_prebuild_plugin_failure(docker_tasker, required):
+    workflow = DockerBuildWorkflow(SOURCE, "test-image")
+    assert workflow.plugin_failed is False
+    runner = PreBuildPluginsRunner(docker_tasker, workflow,
+                                   [{"name": "dockerfile_content",
```

Comment:
How about using a name that makes it clear the plugin doesn't exist, like `"name": "no_such_plugin"`?